app/test_main.py

```diff
@@ -1,3 +1,47 @@
 from app.main import get_human_age
 
-# write your code here
+import pytest
+
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_res", [
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="under_lower_boundary_condition"
+            ),
+            pytest.param(
+                15,
+                23,
+                [1, 1],
+                id="condition_for_return_1"
```

Comment:
The description should describe tests, your description is more appropriate for the title.